cors/middleware.go

```diff
@@ -0,0 +1,13 @@
+package cors
+
+import "net/http"
+
+func Middleware(handler http.Handler) http.Handler {
```

Comment:
This is awesome, basically it is the way things work behind the curtains. Really cool